content/java/concepts/arrays/arrays.md

```diff
@@ -1,105 +1,83 @@
 ---
 Title: 'Arrays'
-Description: 'In Java, an array is used to store a list of elements of the same datatype. Arrays are fixed in size and their elements are ordered. java // Create an array of 5 int elements int[] marks = {10, 20, 30, 40, 50};  In Java, an array can be created in the following ways:'
+Description: 'Arrays are used for storing elements of the same data type in a fixed collection, ordered by insertion.'
 Subjects:
+  - 'Code Foundations'
   - 'Computer Science'
 Tags:
   - 'Arrays'
   - 'Data Types'
+  - 'Data Structures'
 CatalogContent:
   - 'learn-java'
   - 'paths/computer-science'
 ---
 
-In Java, an array is used to store a list of elements of the same datatype.
+In Java, an array is used for storing elements of the same data type in a fixed collection, ordered by insertion.
```

Comment:
Can we clarify what fixed means? It can be different from my suggestion.
```suggestion
In Java, an array is used for storing elements of the same data type in a fixed-sized collection, ordered by insertion. Being fixed in size means that unlike with [ArrayList](https://www.codecademy.com/resources/docs/java/array-list), elements cannot be removed or added after the array has been created. They can only be modified.
```